pkg/handlers/healthProbes.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/golang/glog"
+)
+
+// LivenessProbe is used to check if this service is alive.
+func LivenessProbe(w http.ResponseWriter, r *http.Request) {
+	glog.V(2).Info("livenessProbe")
+	fmt.Fprint(w, "OK")
+}
+
+// ReadinessProbe checks if connection to CCX is established - TODO is this right??
```

Comment:
I don't think you need to do anything here.  Readiness makes more sense for services that accept api requests.  Here we are adding this probe just to comply with kube.